### PR TITLE
[Main Course] - 12306ticket

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for 12306ticket

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/One_Day_Build/2021-11-10-12306ticket.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+# 12306智能订票
+
+!!! info
+    作者：Void，发布于2021-11-10，阅读时间：约6分钟，微信公众号文章链接：[:fontawesome-solid-link:](https://mp.weixin.qq.com/s/ctAdVwr1CdzRSPQfrRVGPQ)
+
+## 1 前言
+
+节假日的时候，回家或者热门景点的火车票往往一售而空。除了拼人品、拼手速地蹲点守候，我们还可以用科学的方式智能订票。
```
Comment (Realvincentyuan) [inline docs/Main_Course/One_Day_Build/2021-11-10-12306ticket.md:12]: 往往·很快·一售而空？

**Review by Realvincentyuan — APPROVED** [review]
很有意思，谢谢分享

**Review by Tina0330 — APPROVED** [review]
完美！

File: docs/Main_Course/One_Day_Build/2021-11-10-12306ticket.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Main_Course/One_Day_Build/2021-11-10-12306ticket.md]: 已修改

**Pull request merged**